src/Microsoft.Identity.Client/Features/ConfidentialClient/IByRefreshToken.cs

```diff
@@ -0,0 +1,51 @@
+﻿//------------------------------------------------------------------------------
+//
+// Copyright (c) Microsoft Corporation.
+// All rights reserved.
+//
+// This code is licensed under the MIT License.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files(the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and / or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions :
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+//
+//------------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.Identity.Client
+{
+    /// <summary>
+    /// 
+    /// </summary>
+    public interface IByRefreshToken
+    {
+        /// <summary>
+        /// This method should be used when you have a solution using ADAL 2.x and 
+        /// caching a refresh token, and you want to migrate to MSAL.NET.
+        /// During the migration process, it enables you to store in MSAL.NET token
+        ///  cache an access token and refresh token corresponding to <paramref="refreshToken"/>.
+        /// From there you will be able to use MSAL.NET new API, in particular
+        /// AcquireTokenSilentAsync() which will renew the user token.  
```

Comment:
yeah, then pls. write that instead of user token as that is not clear.

---
In reply to: [246890414](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/759#discussion_r246890414) [](ancestors = 246890414)

Reply:
this seems not to be addressed. (please also see the indentation on each of the lines)

---
In reply to: [246908783](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/759#discussion_r246908783) [](ancestors = 246908783,246890414)